### ISSUE TITLE
Basic math / Manipulating dates and times

### ISSUE THREAD

**Issue by benfishbus** [pipes-digital/pipes#93]
Is there any way to perform arithmetic in Pipes? I want to convert a time from mm:ss to just seconds. Can I call a webservice like https://api.mathjs.org/ ?

**Comment by onli** [comment]
Yes. Well, Pipes does not have a block for this. But you can call https://api.mathjs.org/. You just need to feed your target calculation url into a download block.

If you need to take the mm:ss from a different feed then feeding that into a download block is not obvious, as the download block only has a text input. The trick is to use a foreach block, fill it with a download block, and then connect its input with a feed that contains the URLs you want to call. 

The result will be combined into one feed and the created download calls will also be properly throttled.

I'm not against adding a new block that does calculations if someone comes up with a usable block layout.
